modules/lvms-creating-lvms-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-lvms-cluster-using-web-console_{context}"]
+= Creating a Logical Volume Manager cluster using the web console
+
+You can create a Logical Volume Manager cluster on a worker node using the {product-title} web console.
+
+.Prerequisites
+
+* You have installed {lvms}.
+
+.Procedure
+
+. Log in to {product-title} web console.
+. Click *Operators* -> *Installed Operators* to view all the installed operators.
+. In the `openshift-storage` namespace, click *LVM Storage* -> *LVMCluster* -> *Create LVMCluster*.
+. In the *Create LVMCluster* page, select either *Form view* or *YAML view*.
+. Configure the `LVMCluster` parameters such as `name`, `deviceClasses`, and so on.
+. Click *Create*.
++
+[IMPORTANT]
+====
+You can only create a single instance of the `LVMCluster` custom resource (CR) on an {product-title} cluster.
+====
++
+. Optional: To edit the `LVMCLuster` CR:
+.. Go to the *LVMCluster* tab, and click *Actions* -> *Edit LVMCluster*. 
+.. Click *YAML*, and edit the required parameters in the `LVMCLuster` CR YAML.  
+
+.Verification
+
+. In the {product-title} web console, click *Storage* → *Storage Classes*.
```

Comment:
The user does not need to verify the occurrence of StorageClass. Instead, they can check if the state of LVMCluster is ready.

Reply:
The state is already visible on the same screen right after clicking on `Create`: 

<img width="2153" alt="Screenshot 2024-01-23 at 13 04 16" src="https://github.com/openshift/openshift-docs/assets/5543238/8e940bf9-4bbd-48f7-9c0a-2793dc280b74">